troubleshooting/commonly-occurring-error-conditions.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+[id="commonly-occurring-error-conditions"]
+= Commonly occurring error conditions
+include::modules/common-attributes.adoc[]
+:context: error-conditions
+
+toc::[]
+
+
+[role="_abstract"]
+The vast majority of errors occur during Collector startup, when the Collector configures itself, finds or downloads an appropriate kernel driver for the system, and then starts collecting events. 
```

Comment:
```suggestion
Most errors occur during Collector startup when Collector configures itself and finds or downloads a kernel driver for the system. 
```

I am not sure about `and then starts collecting events.` does error occur during this phase?

Reply:
Errors that occur after Collector has begun collecting events are more difficult to generally troubleshoot (and are somewhat outside of the scope of these docs), so I agree that this should be removed from this paragraph.

Reply:
ack